tests/pre_merge/utils/metrics/test_aupro.py

```diff
@@ -58,3 +63,10 @@ def test_pro(labels, preds, fpr_limit, aupro):
     assert torch.allclose(computed_aupro, aupro, atol=TOL)
     assert torch.allclose(computed_aupro, ref_pro, atol=TOL)
     assert torch.allclose(aupro, ref_pro, atol=TOL)
+
+    binned_pro = AUPRO(fpr_limit=fpr_limit, thresholds=thresholds)
```

Comment:
Could you put this in a separate test for the sake of keeping them independent?

Reply:
sure !

Reply:
done

Reply:
I have also renamed some pro to aupro ;-)